.mailmap

```diff
@@ -44,8 +45,10 @@ Ben Noordhuis <info@bnoordhuis.nl> <bnoordhuis@bender.(none)>
 Ben Noordhuis <info@bnoordhuis.nl> <ben@strongloop.com>
 Ben Taber <ben.taber@gmail.com>
 Benjamin Coe <bencoe@gmail.com> <ben@npmjs.com>
+Benjamin Coe <bencoe@gmail.com> <bencoe@google.com>
```

Comment:
@bcoe Is the gmail one your preferred one? That’s what this is assuming here but please correct me if I’m wrong.